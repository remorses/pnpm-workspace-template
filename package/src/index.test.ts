import { test, expect } from 'vitest'

test('ready', () => {
    expect(true).toBe(true)
    expect({ x: 3 }).toMatchSnapshot()
})

test('two', () => {
    expect(true).toBe(true)
    console.log('hello')
    expect({ x: 3 }).toMatchSnapshot()
})


