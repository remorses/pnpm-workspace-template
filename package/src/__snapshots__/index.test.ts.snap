// Vitest Snapshot v1

exports[`ready 1`] = `
{
  "x": 3,
}
`;

exports[`two 1`] = `
{
  "x": 3,
}
`;
